### PR TITLE
PLAT-141503: DateTimePicker - Active bar has no background color

### PR DESCRIPTION
**Pull request by paul-beldean-lgp**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
DateTimePicker active bar was transparent on Carbon, Copper, Cobalt and Titanium skins

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fixed active bar color for Carbon, Copper, Cobalt and Titanium skins

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
PLAT-141503

### Comments

Enact-DCO-1.0-Signed-off-by: Paul Beldean paul.beldean@lgepartner.com

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ The following is a curated list of changes in the Enact agate module, newest cha
 ### Fixed
 
 - `agate/Checkbox`, `agate/FanSpeedControl`, `agate/ImageItem`, `agate/Item`, and `agate/WindDirectionControl` to match latest design for Silicon skin
+- `agate/DateTimePicker` active bar background color
```
Comment (daniel-stoian-lgp) [inline CHANGELOG.md:30]: please specify which skins are fixed with this change

File: styles/colors-carbon.less [hunk 2]
```diff
@@ -64,6 +64,10 @@
 @agate-colorpicker-bg-color: rgba(0, 0, 0, 0.5);
 @agate-colorpicker-palette-shadow: @agate-popup-shadow;
 
+// DateTimePicker
+// ---------------------------------------
+@agate-datetimepicker-active-container-bg-color: @agate-accent;
```
Comment (daniel-stoian-lgp) [inline styles/colors-carbon.less:69]: These can be optimized:  Set @agate-datetimepicker-active-container-bg-color: @agate-accent; in colors-base and remove the duplicate declarations from colors-[skin].less.
For electro skin, you can set it to transparent or inherit

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by seunghoh — COMMENTED** [review]
Thanks for the PR but let's leave as is because
- We had no background color on active bar when we had an original skin(cooper, cobalt etc) as intended
- We don't have any layout broken

**Pull request closed without merging**